docs/src/variables.md

```diff
@@ -65,8 +69,8 @@ julia> y
 decision variable
 ```
 
-Because `y` is a Julia variable, I can bind it to a different value. For example,
-if I go:
+Because `y` is a Julia variable, we can bind it to a different value. For
+example, if we go:
```

Comment:
"if we go" sounds a bit too informal and might be confusing for non-native speakers. "if we write" instead?